### DISCUSSION TITLE
StrongName version?

### DISCUSSION THREAD

**Discussion by LazyLeecher** [MoonStorm/FastCrud#174] [Ideas]
Hi all,

Thanks for the project, I use a lot. But recently I need a version with SN and the FastCrud doesn't have one, can you guys compile a new project with SN on nuget like Dapper did, or include in your main project?

Thanks.

**Comment by MoonStorm** [comment]
Sure, we can look into it. 
Since strong naming only offers identity and NOT code integrity, plus nowaydays this practice seems like it's being [discouraged](https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md) or [not even used](https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named) (`"For .NET Core and .NET 5+, strong-named assemblies do not provide material benefits. The runtime never validates the strong-name signature, nor does it use the strong-name for assembly binding."`), why do you need it? I'm simply curious.
